src/components/PomodoroButtons/PomodoroButtons.tsx

```diff
@@ -0,0 +1,19 @@
+import styles from "./PomodoroButtons.module.scss";
+import { FC, useState } from "react";
+import PomodoroButton from "./PomodoroButton/PomodoroButton";
+
+const PomodoroButtons: FC = () => {
+  const [activeButton, setActiveButton] = useState("pomodoro");
```

Comment:
Consider being more strict about types - in this case activeButton type is inferred and is just string. First of all I would create separte file e.g. PomodoroButtons/ButtonTypes.ts with a following content: `export const ButtonTypes = {
    POMODORO: 'pomodoro',
    SHORT_BREAK: 'short break',
    LONG_BREAK: 'long break',
} as const

export type ButtonTypesKey = keyof typeof ButtonTypes;
export type ButtonTypesValue = typeof ButtonTypes[keyof typeof ButtonTypes];` and it could be later used like that: 1) stricter type `const [activeButton, setActiveButton] = useState<ButtonTypesValue>(ButtonTypes.POMODORO);` 2) rendering buttons in a loop to make it less repetitive: `       <div className={styles.buttons}>
            {Object.values(ButtonTypes).map(type => (
                <PomodoroButton key={type} {...{onButtonClick, activeButton, buttonText: type}} />
            ))}
        </div>`

Reply:
But I didn't need this line export type ButtonTypesKey = keyof typeof ButtonTypes;